apps/client/src/app/(auth)/register/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { Button, TextInputIcon } from "@spin-spot/components";
+import { useRouter } from "next/navigation";
+
+export default function Register() {
+  const router = useRouter();
+
+  const handleRegisterClick = () => {
+    router.push("/login");
+  };
+  return (
+    <div className="absolute inset-0 z-40 flex items-center justify-center">
+      <div className="mt-24 w-96 space-y-4 rounded-lg p-8 sm:mt-36">
+        <div className="flex flex-col gap-1">
+          <h2 className="text-neutral mb-1 text-center text-3xl font-black">
+            Registrarse
+          </h2>
+          <TextInputIcon
+            placeholder="example@email.com"
+            type="email"
+            topRightLabel="Correo Electrónico"
+            svg="email"
+          ></TextInputIcon>
```

Comment:
Aqui también auto-cerrar la etiqueta:
```suggestion
          <TextInputIcon
            placeholder="example@email.com"
            type="email"
            topRightLabel="Correo Electrónico"
            svg="email"
          />
```